docker/test/fasttest/run.sh

```diff
@@ -250,6 +250,8 @@ TESTS_TO_SKIP=(
 
     # Look at DistributedFilesToInsert, so cannot run in parallel.
     01460_DistributedFilesToInsert
+
+    01504_rocksdb
```

Comment:
I looked through the test and I don't see why it can't be used in the fasttest, maybe you can add a comment describing why it should not be run?

Reply:
I've comment that, because fasttest use  ENABLE_LIBRARIES=0, so rocksdb is disabled by default.